service-deployment/clusterconfig-example.yaml

```diff
@@ -80,12 +80,13 @@ clusterinfo:
     server_port: 9186
     # secret for signing authentication tokens, e.g., "Hello PAI!"
     jwt_secret: your_jwt_secret
-    # authentication database file path relative to PAI data path
-    lowdb_path: "rest-server/user.db.json"
-    # database admin username
-    lowdb_admin: your_admin_username
-    # database admin password
-    lowdb_passwd: your_admin_password
+    # etcd database admin username
+    etcd_admin: your_admin_username
```

Comment:
>etcd_admin [](start = 4, length = 10)

Better to rename it as pai_admin?

Reply:
Agree with Fan. 'admin' is PAI's admin, not etcd's. And so is the 'password'. Suggest change to:
- `etcd_admin` --> `default_pai_admin_username`, and
- `etcd_passwd` --> `default_pai_admin_password`.